src/app/artwork/artwork.component.css

```
@@ -4,9 +4,13 @@ img{
 
 }
 
-.col-sm{
-    text-align: right;
-    padding: 20px;
+.col{
```

Should you add a space here?
```suggestion
.col {
```